### PR TITLE
[CLApp] Adding Deactivate Element Process

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
**📝 Description**
In this PR I am adding a process to deactivate elements according to a certain `Variable `and `Threshold`

~~~json
{
            "python_module" : "element_deactivation_process",
            "kratos_module" : "KratosMultiphysics.ConstitutiveLawsApplication",
            "process_name"  : "ElementDeactivationProcess",
            "Parameters"    : {
				"model_part_name"             : "Structure.Parts_Solid_Solid_Auto1",
				"variable_name"               : "DAMAGE",
				"variable_maximum_threshold"  : 0.99,
				"average_calculation_over_ip" : true
            }
        }
~~~

I though to implement this in the master based on the work of @atfard in his branch

### CODE REVIEW

**Comment by AlejandroCornejo** [comment]
@atfard you can try to use this and let me know if it works properly.

File: applications/ConstitutiveLawsApplication/python_scripts/element_deactivation_process [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+import KratosMultiphysics as KM
+
+def Factory(settings, Model):
```
Comment (loumalouomega) [inline applications/ConstitutiveLawsApplication/python_scripts/element_deactivation_process]: This could be a C++ process, it would be faster

**Review by loumalouomega — CHANGES_REQUESTED** [review]
Add test + move to C++

**Comment by AlejandroCornejo** [comment]
> Add test + move to C++

Yes indeed, after @atfard tries it, I will move it to parallel c++

**Comment by atfard** [comment]
I checked it in a DCB test with isotropic damage, and is working properly. The point is that in Traction Separation CL, damage variable is vector so we should think about generalizing it.
image

**Review by loumalouomega — DISMISSED** [review]
I applied the suggestions myself to avoid disturb you

**Review by atfard — DISMISSED** [review]

**Review by atfard — APPROVED** [review]

**Pull request merged**